packages/tools/create-kadena-app/templates/angular/src/environments/environment.ts

```diff
@@ -0,0 +1,12 @@
+import { ChainId, ChainwebNetworkId } from '@kadena/types';
+
+// this would normally be your mainnet configuration, but our test contract has only been deployed on testnet
```

Comment:
```suggestion
// this would normally be your mainnet configuration, but our test contract `free.cka-message-store` has only been deployed on testnet
```